docs/tutorials/polynome2/model_run_step_0.py

```diff
@@ -30,6 +30,9 @@ def r2(y_true, y_pred):
 def run():
     global HISTORY
     (x_train, y_train), (x_valid, y_valid) = load_data()
+    
+    if point["activation"] == "identity":
+    	point["activation"] = None
```

Comment:
these lines are unnecessary here because no `point` is passed, an exception will be raised.